pkg/controller/networkpolicy/validate.go

```diff
@@ -492,6 +492,9 @@ func (a *antreaPolicyValidator) validateAppliedTo(ingress, egress []secv1alpha1.
 func (a *antreaPolicyValidator) validatePeers(ingress, egress []secv1alpha1.Rule) (string, bool) {
 	checkPeers := func(peers []secv1alpha1.NetworkPolicyPeer) (string, bool) {
 		for _, peer := range peers {
+			if peer.NamespaceSelector != nil && peer.Namespaces != nil {
```

Comment:
Should we validate "Self" (or maybe Namespaces) cannot be set for AppliedTo?

Reply:
Thanks for the reminder! Added validation and e2e test

Reply:
Actually just realized that this will be part of openapi validation, so if you specified `namespaces` for appliedTo, or even for Antrea namespaced policies, the validation will already fail. If the user force to skip validation. this field will get dropped.